awx/api/filters.py

```diff
@@ -283,6 +283,9 @@ def filter_queryset(self, request, queryset, view):
                         search_value, new_keys, _ = self.value_to_python(queryset.model, key, force_text(value))
                         assert isinstance(new_keys, list)
                         search_filters[search_value] = new_keys
+                    # by definition, search *only* joins across relations,
+                    # so it _always_ needs a .distinct()
+                    needs_distinct = True
                     continue
```

Comment:
@AlanCoding the issue is that the `__search` continues to the next loop iteration without hitting the code *below* that properly sets `needs_distinct = True` for this filter

Reply:
https://github.com/ansible/awx/pull/5749/files#diff-515cd3f41f08cc9dd95b3f4caa1525deR312